modules/installation-configuration-parameters.adoc

```diff
@@ -224,6 +229,24 @@ endif::[]
 ifndef::bare,ibm-power,ibm-z,ash[]
 Before you deploy an {product-title} cluster, you provide parameter values to describe your account on the cloud platform that hosts your cluster and optionally customize your cluster's platform. When you create the `install-config.yaml` installation configuration file, you provide values for the required parameters through the command line. If you customize your cluster, you can modify the `install-config.yaml` file to provide more details about the platform.
 endif::bare,ibm-power,ibm-z,ash[]
+ifdef::aws-outposts[]
+
+When deploying {product-title} cluster with remote workers running in AWS Outposts, some customization steps are required in order to provide the specific details of the Outposts environment.
+
+[NOTE]
+====
+The AWS Outposts environments has the following limitations which require manual modification of the
+`install-config.yaml` file:
+
+* Unlike AWS Regions, which offer near-infinite scale, AWS Outposts are limited by their provisioned capacity, EC2 family and generations, configured instance sizes, and availability of compute capacity that is not already consumed by other workloads. Therefore, when creating new {product-title} cluster, you need to provide the supported instance type in the `compute.platform.aws.type` section in the configuration file.
+
+* When deploying {product-title} cluster with remote workers running in AWS Outposts, only one subnet can be used for the compute instances - the subnet in which the Outpost instance was created in. Therefore, when creating new {product-title} cluster, it recommended to provide the relevant Availability Zone in the `compute.platform.aws.zones` section in the configuration file, in order to limit the compute instances to this Availability Zone.
+
+* Amazon Elastic Block Store (EBS) gp3 volumes aren't supported by the AWS Outposts service. This volume type is the default type used by the {product-title} cluster. Therefore, when creating new {product-title} cluster, you must change the volume type in the `compute.platform.aws.rootVolume.type` section to gp2.
```

Comment:
Is it relevant to mention other types of storage that aren't supported on outposts, either?

Reply:
The gp3 storage type is mentioned here because this is the default storage type used by AWS platform, so it need special configuration in case of Outposts
I don't think listing all other not supported types is required here